coa/pkg/apis/v1alpha2/types.go

```diff
@@ -135,6 +136,29 @@ const (
 	TargetPropertyNotFound State = 12000
 )
 
+func GetHttpStatus(code int) State {
+	switch code {
+	case 200:
+		return OK
+	case 202:
+		return Accepted
+	case 400:
+		return BadRequest
+	case 403:
+		return Unauthorized
+	case 404:
+		return NotFound
+	case 405:
+		return MethodNotAllowed
+	case 409:
+		return Conflict
+	case 500:
+		return InternalError
+	default:
+		return None
```

Comment:
None is not that ideal. It doesn't carry any http status information. If httpstatus code is 4xx or 5xx, but not defined in COAError. I think it is still worthy to keep them. @linyguo , @iwangjintian what do you think?